Extensions/find_inactives.js

```diff
@@ -290,7 +307,7 @@ XKit.extensions.find_inactives = new Object({
 
 		}
 
-		$(".find-inactives-blog").click(function() {
+		$(".find-inactives-blog-link").click(function() {
```

Comment:
I don't see the problem then, I guess it would have opened a new tab when you clicked unfollow otherwise. Personally I always clicked on the blog name anyways